packages/notifier/src/notifier.js

```diff
@@ -1,32 +1,31 @@
 /// <reference types="ses"/>
 
 import { assert } from '@agoric/assert';
-import { makePromiseKit } from '@endo/promise-kit';
-import { E } from '@endo/eventual-send';
-import { Far } from '@endo/marshal';
-import { makeAsyncIterableFromNotifier } from './asyncIterableAdaptor.js';
+import { E, Far } from '@endo/far';
 
 import './types-ambient.js';
+import { makePublishKit } from './publish-kit.js';
+import { subscribeLatest } from '../tools/subscribe.js';
 
 /**
  * @template T
  * @param {ERef<BaseNotifier<T> | NotifierInternals<T>>} sharableInternalsP
```

Comment:
Can the `makeNotifier` argument type be simplified as well?
```suggestion
 * @param {ERef<LatestTopic<T>>} sharableInternalsP
```